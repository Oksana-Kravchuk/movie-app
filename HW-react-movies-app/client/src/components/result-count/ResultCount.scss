@import '../../common/styles/colours';

.results-count {
  display: flex;
  font-size: 18px;
  flex-wrap: wrap;
  margin-bottom: 18px;

  &__main {
    font-weight: bold;
    margin-right: 5px;
  }

  &::before {
    display: block;
    content: '';
    width: 100%;
    height: 3px;
    margin-bottom: 18px;
    background-color: $grey;
    border-bottom: 1px solid $black;
  }
}
