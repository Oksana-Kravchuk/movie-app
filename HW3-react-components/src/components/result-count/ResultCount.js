import React from 'react';
import PropTypes from 'prop-types';
import './ResultCount.scss';

const ResultCount = ({ amount }) => {
  return (
      <div className="results-count">
        <div className="results-count__main">{amount}</div>
        <div>movies found</div>
      </div>
  )
}

ResultCount.propTypes = {
  amount: PropTypes.number
}

export default ResultCount;