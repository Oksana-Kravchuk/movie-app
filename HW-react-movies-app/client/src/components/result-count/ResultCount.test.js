import React from 'react';
import Enzyme from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ResultCount from './ResultCount';

Enzyme.configure({ adapter: new Adapter() });

describe('ResultCount ', () => {
  let wrapper;

  beforeEach(() => {
    const props = {
     amount: 10
    };
    wrapper = shallow(<ResultCount {...props} />);
  });

  it('render correctly ResultCount component', () => {  
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});