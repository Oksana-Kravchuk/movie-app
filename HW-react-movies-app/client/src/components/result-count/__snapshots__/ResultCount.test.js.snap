// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultCount  render correctly ResultCount component 1`] = `
<div
  className="results-count"
>
  <div
    className="results-count__main"
  >
    10
  </div>
  <div>
    movies found
  </div>
</div>
`;
